db/migrate/20230914064811_add_unique_index_on_blob_checksum.rb

```diff
@@ -12,7 +12,7 @@ def change
         UPDATE storage_tables_blobs
         SET attachments_count = attachments_count + 1,
             attachments_count_modified = CURRENT_TIMESTAMP
-        WHERE checksum = NEW.checksum;
+        WHERE partition_key = NEW.blob_key AND checksum = NEW.checksum;
```

Comment:
Hebben we ook een beveiliging dat de partition_key en checksum niet geupdate mogen worden? Want als dat wel kan gebeuren dan kloppen de counts niet meer.

Reply:
Daar is nu een trigger voor toegevoegd.